elpis/transformer/elan.py

```diff
@@ -171,6 +171,8 @@ def import_eaf_file(eaf_paths, context, add_annotation, tmp_dir):
     special_cases = set(context['special_cases'].splitlines())
     translation_tags = set(context['translation_tags'].splitlines())
 
+    reset_annotations()
```

Comment:
This is just a question, doesn't necessarily need to change, but what does this do? (should be made clearer when the docstring is updated, but "resetting the annotations" didn't make a whole lot of sense to me here).

Reply:
Added a description to the docstring.